docs/index.rst

```diff
@@ -72,6 +72,8 @@ contacting support.
 +--------------------------------------------------+---------------+----------------+
 | :ref:`falcon`                                    | >= 1.0        | No             |
 +--------------------------------------------------+---------------+----------------+
+| :ref:`fastapi`                                   | >= 0.63       | Yes            |
```

Comment:
We can probably support older versions of fastapi since I assume/hope the API for adding middleware hasn't changed much